test/type_test.js

```diff
@@ -0,0 +1,86 @@
+goog.require('plain');
+
+describe('The plain.type function', function () {
+  /**
+   * Assert that plain.type identifies something as a certain type.
+   * @param value The javascript object to identify.
+   * @param expected The value we explect plain.type to return.
+   * @param context A message describing the value.
+   */
+  function assertType(value, expected, context) {
+    expect(plain.type(value))
+        .withContext('identifies ' + context + ' as ' + expected).toBe(expected);
```

Comment:
Please use `identifies ${context} as ${expected}` for the formatted strings

Reply:
Done